test/initialization-tests.html

```diff
@@ -139,14 +139,18 @@
           'attached-before',
           'ready-before',
           // TODO: These tests are failing with Polymer 2 (#95).
-          // 'default-after', 'attached-after', 'ready-after',
-          // 'default-below', 'attached-below', 'ready-below',
+          // 'default-below',
+          // 'default-after',
+          // 'attached-below',
+          // 'attached-after',
+          // 'ready-below',
+          // 'ready-after',
+          // 'ready-container',
           'default-above',
           'attached-above',
           'ready-above',
           'default-container',
           'attached-container',
-          'ready-container'
```

Comment:
Previously `ready-container` was never defined, so that must be why it was failing. Is it still the case?

Reply:
It was never defined so it was passing because the test tests that nothing in the URL changed. It was never defined because I accidentally forgot to change `is: 'ready-above'` to `is: 'ready-container'`. I then fixed this and it was properly defined and found that the test was failing. I assume this is because of the change to the data-binding system in Polymer 2 compared to Polymer 1.